Makefile-az.mk

```diff
@@ -22,7 +22,9 @@ az-login: ## Login into Azure
 	az account set --subscription $(AZURE_SUBSCRIPTION_ID)
 
 az-mkrg: ## Create resource group
```

Comment:
Add a screenshot to the PR description with how it looks now? You mentioned that a `false`/`true` is now output?

Reply:
```
make az-mkrg
if ! az group exists --name bookish-goldfish-r79r65v6x4jhggq; then \
        az group create --name bookish-goldfish-r79r65v6x4jhggq --location westus2 -o none; \
fi
false
```

Reply:
Adding an `@` before the if makes it instead look like this:
```
$ make az-mkrg
false
```

Reply:
I prefer having the if statement showing in the output to show what's being checked there. Gives context to the false

Reply:
Now there's no output,  * how fancy*, ty chatgpt

Reply:
I think, I personally would like which commands are being called, and their output to be displayed.

I think having the whole if block being output feels a little messy, but I'd possibly prefer that over no output. 

My optimal would be:
> az group exists --name $(AZURE_RESOURCE_GROUP)
> [output]
> [if the if block is executed, than output the az group create command, and its output]